project/src/test/java/com/google/sps/UserTest.java

```diff
@@ -30,14 +30,13 @@
   @Test
   public void buildInstanceWithAllFields() {
     // Create a User with all fields set, and verify that all fields are correctly set
-    User user =
-        new User.Builder(NAME, EMAIL)
-            .setInterests(INTERESTS)
-            .setSkills(SKILLS)
-            .setEventsHosting(EVENTS_HOSTING)
-            .setEventsParticipating(EVENTS_PARTICIPATING)
-            .setEventsVolunteering(EVENTS_VOLUNTEERING)
-            .build();
+    User user = new User.Builder(NAME, EMAIL)
```

Comment:
Ditto with taking out these changes, I think the original satisfied the linter (and the new version satisfies prettier), but the left would conform to the rest of our linted changes

Reply:
done, yes i did a 'make pretty'